### PR TITLE
Bavet121 desc

### PR DESCRIPTION
**Pull request by MassimoVilla**
Here comes BAVet121, a Dǝrsāna Mikāʾel followed by texts containing magic names

### CODE REVIEW

File: VaticanBAV/et/BAVet121.xml [hunk 1]
```diff
@@ -0,0 +1,562 @@
...
+                                </collation>
+                             </supportDesc>
+                            <layoutDesc>
+                                <layout columns="2" writtenLines="22"/>
+                            </layoutDesc>
+                        </objectDesc>
+                        
+                        <handDesc>
+                            <handNote xml:id="h1" script="Ethiopic">
+                                <seg type="script">19th-century script</seg>
```
Comment (CarstenHoffmannMarburg) [inline VaticanBAV/et/BAVet121.xml:463]: 19th century script

Comment (eu-genia) [reply VaticanBAV/et/BAVet121.xml:463]: why? it is correct with the dash

File: VaticanBAV/et/BAVet121.xml [hunk 2]
```diff
@@ -0,0 +1,562 @@
...
+                                <locus from="7v" to="16v"/>
+                                <title type="complete" ref="LIT1295Dersan#Hedar"/>
+                                <msItem xml:id="ms_i1.2.1">
+                                    <locus from="7v" to="13v"/>
+                                    <title type="complete" ref="LIT1295Dersan#HomilyHedar"/>
+                                    <explicit xml:lang="gez">
+                                        ወበጸሎቱ፡ ለቅዱስ፡ ሚካኤ<supplied reason="omitted">ል</supplied><gap reason="ellipsis" extent="unknown" resp="PRS4805Grebaut PRS9530Tisseran"/> 
+                                        ያድኅኖ፤ አመሣግሪሁ፡ ለሰይጣን፨ ለፍቅሩ፡ ወለጸሐፊሁኒ። ኃጥእ፡ ወአባሲ፡ ገብረ፡ ማርያም፡ ለዓ<supplied reason="omitted">ለ</supplied>መ፡ ዓለም፡ አሜን።
+                                    </explicit>
+                                    <note>The name of the owner <persName role="scribe" ref="PRS14580GabraMa ">Gabra Māryām</persName> is mentioned in the supplication formula at the end
```
Comment (eu-genia) [inline VaticanBAV/et/BAVet121.xml:54]: remove space before the closing` "`

File: VaticanBAV/et/BAVet121.xml [hunk 3]
```diff
@@ -0,0 +1,562 @@
...
+                                <date notBefore="1800" notAfter="1899"/>
+                                <seg type="rubrication">Headings.</seg>
+                            </handNote>
+                        </handDesc>
+                        
+                        <additions>
+                            <list>
+                                <item xml:id="a1">
+                                    <desc type="Supplication">
+                                        The note is a short rhymed supplication written down by the scribe of the main text, <persName role="scribe" ref="PRS14580GabraMa ">Gabra Māryām</persName>.
```
Comment (eu-genia) [inline VaticanBAV/et/BAVet121.xml:473]: remove space before "

File: VaticanBAV/et/BAVet121.xml [hunk 4]
```diff
@@ -0,0 +1,562 @@
...
+                        </adminInfo>
+                    </additional>
+                    
+                </msDesc>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <t:profileDesc xmlns:t="http://www.tei-c.org/ns/1.0">
```
Comment (eu-genia) [inline VaticanBAV/et/BAVet121.xml:534]: remove t: and xmlns

File: VaticanBAV/et/BAVet121.xml [hunk 5]
```diff
@@ -0,0 +1,562 @@
...
+            <creation/>
+            <t:abstract>
```
Comment (eu-genia) [inline VaticanBAV/et/BAVet121.xml:536]: remove t: throughout

**Review by eu-genia — CHANGES_REQUESTED** [review]
thank you, wonderful!

**Review by CarstenHoffmannMarburg — APPROVED** [review]

File: VaticanBAV/et/BAVet121.xml [hunk 6]
```diff
@@ -0,0 +1,558 @@
...
+                            
+                            <msItem xml:id="ms_i1.2">
+                                <locus from="7v" to="16v"/>
+                                <title type="complete" ref="LIT1295Dersan#Hedar"/>
+                                <msItem xml:id="ms_i1.2.1">
+                                    <locus from="7v" to="13v"/>
+                                    <title type="complete" ref="LIT1295Dersan#HomilyHedar"/>
+                                    <explicit xml:lang="gez">
+                                        ወበጸሎቱ፡ ለቅዱስ፡ ሚካኤ<supplied reason="omitted">ል</supplied><gap reason="ellipsis" extent="unknown" resp="PRS4805Grebaut PRS9530Tisseran"/> 
+                                        ያድኅኖ፤ አመሣግሪሁ፡ ለሰይጣን፨ ለፍቅሩ፡ ወለጸሐፊሁኒ። ኃጥእ፡ ወአባሲ፡ ገብረ፡ ማርያም፡ ለዓ<supplied reason="omitted">ለ</supplied>መ፡ ዓለም፡ አሜን።
```
Comment (karljonaskarlsson) [inline VaticanBAV/et/BAVet121.xml:52]: ለፍቁሩ፡ ?

File: VaticanBAV/et/BAVet121.xml [hunk 7]
```diff
@@ -0,0 +1,558 @@
...
+                              <locus from="16v" to="21r"/>
+                                    <title type="complete" ref="LIT1295Dersan#HomilyMiyazya"/>
+                                    <note>The homily corresponds to <ref type="mss" corresp="BAVet82#p1_i1.7.1"/>, where it is inserted within the section for the month of Miyāzyā.</note>
+                           </msItem>
+                                <msItem xml:id="ms_i1.3.2">
+                              <locus from="21r" to="21v"/>
+                                    <title type="complete" ref="LIT1295Dersan#MiracleHamle"/>
+                              <explicit xml:lang="gez">
+                                  ወወፅአ፡ እምቤተ፡ ክርስቲያን፡ ሕያዎ፡ ዝኩ፡ ብእሲ፨ ወነበረ፡ እንዘ፡ ይት<supplied reason="omitted">ቀ</supplied>ነይ፡ ለሊቀ፤
+                                  መላእክር፡ ሚካኤል<supplied reason="omitted">ል</supplied> እስከ፤ ዕለተ፡ ሞቱ፡ ትንብልናሁ፤
```
Comment (karljonaskarlsson) [inline VaticanBAV/et/BAVet121.xml:86]: መላእክት፡ ?

File: VaticanBAV/et/BAVet121.xml [hunk 8]
```diff
@@ -0,0 +1,558 @@
...
+                                    <title type="complete" ref="LIT1295Dersan#SalamTeqemt"/>
+                                </msItem>
+                            </msItem>
+                        </msItem>
+                        
+                        <msItem xml:id="ms_i2">
+                            <locus from="89r" to="93v"/>
+                            <title type="complete" ref="LIT2835RepCh119"/>
+                            <explicit xml:lang="gez">
+                                አንሥአ፡ ኃ<supplied reason="omitted">ይ</supplied>ለከ፡ ወነዓ፡ አድኅነነ። ሚካኤል፡ ርድአነ፨ በጸሎትከ፡ ተማኅፀነ፨ አርዌ፡ ጊጉይ፡
```
Comment (karljonaskarlsson) [inline VaticanBAV/et/BAVet121.xml:328]: አንሥእ

File: VaticanBAV/et/BAVet121.xml [hunk 9]
```diff
@@ -0,0 +1,558 @@
...
+                        </msItem>
+                        
+                        <msItem xml:id="ms_i5">
+                            <locus from="95r" to="95v"/>
+                            <title type="complete" ref="LIT7126Asmat"/>
+                            <incipit xml:lang="gez">
+                                ያኮስ፡ አሜእ፡ ሌኬ፡ ኤንካዕ፡ ከዜዕ፤ ኮርናዕ፤ ክዮን፤ ኤፔካ፡ ኤሎ፤
+                            </incipit>
+                            <explicit xml:lang="gez">
+                                እሉ፡ ኩብራት፡ ወኅቡዓት፨ አስማቲ<supplied reason="omitted">ሁ፡</supplied> ለእግዚእነ፡ ዘተረክቡ፤ በእደ፡ ቅዱስ፡ ሊቀ፡ መላእክት፡ ሚካኤል፡ መግረሬ፡
```
Comment (karljonaskarlsson) [inline VaticanBAV/et/BAVet121.xml:368]: ኩቡራት

File: VaticanBAV/et/BAVet121.xml [hunk 10]
```diff
@@ -0,0 +1,558 @@
...
+                                        ለዛቲ፡ መጽሐፍ፡ ገብረ፡ ማርያም፡ ጸሐፋ፨ ክነፊሁ፡ ለሊቀ፡ መላእክት፡ ይኩኖ፡ ሐገፋ፨ <gap reason="ellipsis" extent="unknown" resp="PRS4805Grebaut PRS9530Tisseran"/>
+                                    </q>
+                                 </item>
+                                <item xml:id="a2">
+                                    <desc type="Supplication">
+                                        The note includes the name of the scribe of the main text, <persName role="scribe" ref="PRS14580GabraMa">Gabra Māryām</persName>. A blank space has been left empy for another name, possibly that of the owner or the patron of the book.
+                                    </desc>
+                                    <locus target="#46r"/>
+                                    <q xml:lang="gez">
+                                        ለፍቅሩ፡ <space reason="owner" extent="unknown"/> ወዓዲ፡ ለጸሐፊሁ፡ ለዝንቱ፡ መጽሐፍ፡ ኃጥእ፡ ወአባሲ፡ ገብረ፡ ማርያም፡ በእንተ፡ ቅዱስ፡ ሚካኤል፡
```
Comment (karljonaskarlsson) [inline VaticanBAV/et/BAVet121.xml:486]: `<space reason="owner" extent="unknown"/>` – this encoding inspires me! Thanks!

File: VaticanBAV/et/BAVet121.xml [hunk 11]
```diff
@@ -0,0 +1,558 @@
...
+                                    </q>
+                                 </item>
+                                <item xml:id="a2">
+                                    <desc type="Supplication">
+                                        The note includes the name of the scribe of the main text, <persName role="scribe" ref="PRS14580GabraMa">Gabra Māryām</persName>. A blank space has been left empy for another name, possibly that of the owner or the patron of the book.
+                                    </desc>
+                                    <locus target="#46r"/>
+                                    <q xml:lang="gez">
+                                        ለፍቅሩ፡ <space reason="owner" extent="unknown"/> ወዓዲ፡ ለጸሐፊሁ፡ ለዝንቱ፡ መጽሐፍ፡ ኃጥእ፡ ወአባሲ፡ ገብረ፡ ማርያም፡ በእንተ፡ ቅዱስ፡ ሚካኤል፡ 
+                                        መልአከ፡ ቅዳሴ። በረከቱ፡ ይኅድር፡ ላዕሌነ፨ በከመ፡ ኀደረ' በረከተ፤ ያዕቆብ፤ ላዕለ፡ ኤፍሬም፡ ወምናሴ፡ ለዓለመ፤ ዓለም፡ አሜን፨
```
Comment (karljonaskarlsson) [inline VaticanBAV/et/BAVet121.xml:487]: " ኀደረ' ", is this as it should be?

**Review by karljonaskarlsson — CHANGES_REQUESTED** [review]

File: VaticanBAV/et/BAVet121.xml [hunk 12]
```diff
@@ -0,0 +1,558 @@
...
+                    
+                    <msContents>
+                        <summary/>
+                        <msItem xml:id="ms_i1">
+                            <locus from="5r" to="89r"/>
+                            <title type="complete" ref="LIT1295Dersan">Dǝrsāna Mikāʾel</title>
+                            <textLang mainLang="gez"/>
+                            
+                            <msItem xml:id="ms_i1.1">
+                           <locus from="5r" to="4v"/>
```
[the remark this answers is not in the document]
Comment (MassimoVilla) [reply VaticanBAV/et/BAVet121.xml:39]: Completely right, thank you

File: VaticanBAV/et/BAVet121.xml (hunk 11 above)
Comment (MassimoVilla) [reply VaticanBAV/et/BAVet121.xml:487]: ኀደረ፤ thank you!

**Review by nafisa-valieva — APPROVED** [review]

**Review by karljonaskarlsson — APPROVED** [review]

**Review by eu-genia — APPROVED** [review]

**Pull request merged**